x/launch/keeper/msg_server_request_remove_validator_test.go

```diff
@@ -0,0 +1,89 @@
+package keeper
+
+import (
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/stretchr/testify/require"
+	"github.com/tendermint/spn/testutil/sample"
+	"github.com/tendermint/spn/x/launch/types"
+)
+
+func TestMsgRequestRemoveValidator(t *testing.T) {
+	var (
+		addr1                     = sample.AccAddress()
+		addr2                     = sample.AccAddress()
+		addr3                     = sample.AccAddress()
+		k, _, srv, _, sdkCtx, cdc = setupMsgServer(t)
+		ctx                       = sdk.WrapSDKContext(sdkCtx)
+		chains                    = createNChain(k, sdkCtx, 4)
+	)
+	tests := []struct {
+		name string
+		msg  types.MsgRequestRemoveValidator
+		want uint64
+		err  error
+	}{
+		{
+			name: "invalid chain",
+			msg: types.MsgRequestRemoveValidator{
+				ChainID: "invalid_chain",
```

Comment:
```suggestion
				ChainID: "invalid_chain",
```

Can we use `sample.ChainID(0)` to ensure it fails when we use a valid chainID format